_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
+- A [Checkstyle](https://checkstyle.sourceforge.io/) rule was added to show an error when a local variable which could be `final` wasn't declared as such. ([@XiangRongLin](https://github.com/XiangRongLin))
 
-- A library we use, PrettyTime, has been updated to a newer version so that a workaround can be removed from NewPipe's code. We get a performance improvement as a bonus! ([@B0pol](https://github.com/B0pol))
+- A library we use, PrettyTime, has been updated to a newer version so that a workaround can be removed from NewPipe's code. The performance improvement is a bonus! ([@B0pol](https://github.com/B0pol))
 
-- Numbers and uppercase letters are now allowed in the application ID. ([@Stypox](https://github.com/Stypox))
+- Numbers and uppercase letters are now allowed in the app ID. ([@Stypox](https://github.com/Stypox))
 
-- Contributors need to discuss their contributions now prior to opening a PR. ([@gkeegan](https://github.com/gkeegan))
+- Contributors need to discuss their contributions prior to opening a PR now. ([@gkeegan](https://github.com/gkeegan))
 	
 - Some deprecations as well as a lot of code cleanup happened. ([@TacoTheDank](https://github.com/TacoTheDank))
 
 - The code now prevents the YouTube website from accessing the available Java packages from NewPipe's parser. This removes a possible attack surface. ([@wb9688](https://github.com/wb9688))
 
-## Where to get this brand-new version
+## Where does one get outfitted with the latest in NewPipe?
 
-NewPipe notifies you about new versions. You can download them when you press the notification. An alternative is the [GitHub release page](https://github.com/TeamNewPipe/NewPipe/releases). If you use the [F-Droid app](https://f-droid.org/), it notifies you as well about an update for [NewPipe](https://f-droid.org/packages/org.schabi.newpipe/) - please keep in mind that it can take F-Droid a while to update their repository. If you have problems installing you may need to uninstall NewPipe then reinstall (make sure to backup data).
+NewPipe notifies you of new versions in a notification you can click to download it. An alternative is the [GitHub release page](https://github.com/TeamNewPipe/NewPipe/releases). If you use the [F-Droid app](https://f-droid.org/), you will be notified of all app news, including [NewPipe](https://f-droid.org/packages/org.schabi.newpipe/). Please keep in mind it can take F-Droid a while to update their repository. If you have problems installing you may need to uninstall NewPipe, and then reinstall it. (Make sure to back up you data.)
 
 If you already had NewPipe installed through F-Droid's repository, to get this version of NewPipe you can do one of the following:
-- Wait for them to update or
-- Switch to the NewPipe repository by following the [directions in the announcement]({% post_url 2020-05-30-f-droid-repo %}) (if you had previously installed NewPipe from GitHub releases you will not have to uninstall NewPipe to switch)
```

Comment:
Meant to know what a repo is without context. Second clause too long.

Reply:
If someone knows what F-Droid is, it is safe to assume they know what a repo is.

Reply:
No, not at all. Also, why not go the other way and explain in plain English how things work instead?